apps/alerts/lib/priority.ex

```diff
@@ -16,7 +16,19 @@ defmodule Alerts.Priority do
     :low
   end
 
-  def priority(%{effect: :delay}, _) do
+  def priority(
+        %{effect: :delay, severity: severity, informed_entity: informed_entities},
+        _time
+      )
+      when severity < 5 do
+    # delay alerts for bus routes under severity 5 are low priority
```

Comment:
This logic doesn't quite seem to capture what's written in the task: "Bus delay alerts due to traffic with a severity of 5 or less are low priority."

Expanding on that, it should also be the case that:
- Bus delay alerts with a `severity` of 6 or more are high priority, regardless of `cause`.
- Bus delay alerts with any `cause` other than "TRAFFIC" and a `severity` of 5 or less are high priority, including alerts without a `cause` value.

So I think we'd need to change the `<` to `<=` and add logic that checks the `cause` of the alert. (I'm no Elixir expert, but at a glance it also looks like we'd need to start adding `cause` to the Alert struct to do so?)

Reply:
Sorry about that, I somehow misread "5 or under" as "less than 5".

I will look into adding the `cause` logic.

Reply:
@lund5000  If I'm reading the GTFS docs right, TRAFFIC isn't a possible cause that can be listed for an alert. It can be listed on a stop time update, but I don't think we currently have access to those in DotCom. 

https://github.com/google/transit/blob/master/gtfs-realtime/spec/en/reference.md#enum-cause

Reply:
That's true, but we regularly publish alerts with all sorts of `cause` values that aren't compliant with that spec: "TRAFFIC", "SPECIAL_EVENT", etc.

Reply:
I wonder if we're using the cause_description there